### PR TITLE
added wishlist page

### PR DESCRIPTION
**Pull request by Samarth-5109**
Screenshot (40)

### CODE REVIEW

File: styles/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  margin-top: 10px;
+}
+.price {
+  color: red;
+  margin-top: 2rem;
+  font-size: 1.3rem;
+}
+footer {
+  display: flex;
+  justify-content: space-between;
```
Comment (GeetaPogul) [inline styles/wishlist.css:50]: One suggestion is don't use html tag names for styles

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <!-- <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up/2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous"/> -->
```
Comment (GeetaPogul) [inline pages/wishlist.html:13]: if you don't need this. remove it.

**Review by GeetaPogul — APPROVED** [review]

File: pages/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+              <i class="fas fa-heart card-icons"></i>
+            </footer>
+          </div>
+  
+          <!-- card-2 -->
+  
+          <div class="basic-card">
+            <img
+              src="https://m.media-amazon.com/images/I/518lp33R1vL.jpg"
+              alt=""
```
Comment (Sharmaryan) [inline pages/wishlist.html:93]: alt text is empty

**Review by Sharmaryan — APPROVED** [review]

File: styles/wishlist.css (hunk 1 above)
Comment (Samarth-5109) [reply styles/wishlist.css:50]: ok

File: pages/wishlist.html (hunk 3 above)
Comment (Samarth-5109) [reply pages/wishlist.html:93]: Oh! will add to it, thanks

**Pull request merged**